charts/opentelemetry-demo/values.yaml

```diff
@@ -762,14 +762,14 @@ opentelemetry-collector:
       pipelines:
         traces:
           processors: [memory_limiter, resource, transform, batch]
-          exporters: [otlp, debug, spanmetrics]
+          exporters: [otlp, spanmetrics]
```

Comment:
We should keep the debug exporter for traces, as well as for metrics and logs. With it the user can see that data is being exported successfully from the collector.

Reply:
```suggestion
          exporters: [otlp, debug, spanmetrics]
```

Reply:
ack, I'll restore de debug exporter.

Reply:
Debug exporter restored